client/components/happychat/style.scss

```diff
@@ -52,6 +52,10 @@
 	right: 12px;
 	font-size: 14px;
 	display: none;		/* disabled */
+	box-sizing: border-box;
+
+	transform: translateZ(0);
```

Comment:
That is indeed for forcing the animation onto the GPU. Do we have any transpiling of `will-change` for older browsers?